### PR TITLE
[Chore] - Fixed Code Block Colors

### LINKED ISSUE #120: Bug: The Rust codes are not colorized
### Describe the bug.

Such as the Rust code in [the pages](https://wasmedge.org/docs/develop/rust/hello_world), the code blocks are not colorized.
As the previous [docs in mdbook](https://wasmedge.org/book/en/sdk/rust/say_hello.html), it should be colorized.

### Describe the steps to reproduce the behavior.

_No response_

### Expected behavior.

_No response_

### Screenshots.

_No response_

### Operating Environment

MacOS

### Additional Information

_No response_

### PR DESCRIPTION
**Pull request by adithyaakrishna**
**Description:**

- Fixes #120

### CODE REVIEW

**Comment by alabulei1** [comment]
Hello, I am a [code review bot](https://github.com/flows-network/github-pr-summary/) on [flows.network](https://flows.network/).

It could take a few minutes for me to analyze this PR. Relax, grab a cup of coffee and check back later. Thanks!

**Comment by adithyaakrishna** [comment]
@alabulei1 Could you please review this PR?

File: docusaurus.config.js [hunk 1]
```diff
@@ -210,6 +210,7 @@ const config = {
             prism: {
                 theme: lightCodeTheme,
                 darkTheme: darkCodeTheme,
+                additionalLanguages: ['rust', 'bash', 'typescript', 'csharp', 'lua', 'protobuf', 'powershell', 'toml', 'yaml'],
```
Comment (q82419) [inline docusaurus.config.js:213]: Please also add the `wasm`, `javascript`, and `go`, thanks.

**Review by q82419 — CHANGES_REQUESTED** [review]

File: docusaurus.config.js (hunk 1 above)
Comment (adithyaakrishna) [reply docusaurus.config.js:213]: Okay sure :)

**Review by q82419 — APPROVED** [review]
LGTM!

**Pull request merged**